v20.2/architecture/replication-layer.md

```diff
@@ -79,6 +79,21 @@ A table's meta and system ranges (detailed in the [distribution layer](distribut
 
 However, unlike table data, system ranges cannot use epoch-based leases because that would create a circular dependency: system ranges are already being used to implement epoch-based leases for table data. Therefore, system ranges use expiration-based leases instead. Expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+#### How leases are transferred from a dead node
+
+When a node disconnects, the process by which each of its leases is transferred to a healthy node is as follows:
+
+1. The cluster waits 9 seconds for the dead node's liveness record, which is stored in a system range, to expire.
+2. A healthy node attempts to acquire the lease. This is rejected because lease acquisition can only happen on the Raft leader, which the healthy node is not (yet).  Therefore, an election must be held.
+3. The rejected attempt at lease acquisition unquiesces the system range holding the liveness record.
+4. What happens next depends on whether the lease is on [table data](#epoch-based-leases-table-data) or [meta ranges or system ranges](#expiration-based-leases-meta-and-system-ranges):
+    - If the lease is on [table data](#epoch-based-leases-table-data), the node that unquiesced the range waits out the Raft election timeout (3 seconds) before kicking off a campaign to try and win Raft leadership so it can become the leaseholder.
+    - If the lease is on [meta or system ranges](#expiration-based-leases-meta-and-system-ranges), the "is the leader alive?" check described above is skipped, since it would introduce a circular dependency on the liveness record used for table data, which is itself stored in a system range.
+5. The Raft election is held and a new leader is chosen from among the healthy nodes.
+6. The lease transfer can now be processed by the newly elected Raft leader.
+
+This process should take no more than 15 seconds.
```

Comment:
The process should be 
 - Liveness expiration (max of 9s)
 - Raft leader election (this should be 1 network roundtrip)
 - Lease transfer, this is a raft proposal (a proposal is typically committed in 1 round trip)

So this should be 9s + 2 RT, @nvanbenschoten does that math check out?